parts/base.js

```diff
@@ -93,4 +93,13 @@ module.exports = {
 			ignoreGlobals: true,
 		}],
 	},
+	overrides: [
+		{
+			// Allow "unpublished" == devDependencies to be imported in config files
+			files: ['./*.config.*'],
```

Comment:
Fancy!!

Reply:
We could also do that for folders like cypress? :thinking: 
Any other idea?

Reply:
Maybe include `tests` for jest files?

`files: ['./*.config.*', 'cypress/**/*', 'tests/**/*']`

Reply:
`__tests__` to then. `__mocks__` should most likely be included too